### PR TITLE
[extractor] added ByteShare

### PR DESCRIPTION
**Pull request by LiJu09**
Added ByteShare extractor.

### CODE REVIEW

File: app/src/main/java/com/lagradost/cloudstream3/extractors/ByteShare.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    override val mainUrl = "https://byteshare.net"
+    override val requiresReferer = false
+
+    override suspend fun getUrl(url: String, referer: String?): List<ExtractorLink> {
+        val sources = mutableListOf<ExtractorLink>()
+        with(app.get(url).document) {
+            this.select("script").map { script ->
+                if (script.data().contains("'use strict';")) {
+                    val data = script.data()
+                        .substringAfter("sources: [").substringBefore("]")
```
Comment (LagradOst) [inline app/src/main/java/com/lagradost/cloudstream3/extractors/ByteShare.kt:17]: you sure you cant parse as json or smth?

Comment (LiJu09) [reply app/src/main/java/com/lagradost/cloudstream3/extractors/ByteShare.kt]: found even better way

**Comment by LagradOst** [comment]
damn, LGTM

**Comment by LagradOst** [comment]
assuming the download link is a raw link

**Comment by LiJu09** [comment]
yeah, its same as in `src: ...`

**Comment by LagradOst** [comment]
> yeah, its same as in `src: ...`

but can you play that link in vlc for example

**Comment by LiJu09** [comment]
> but can you play that link in vlc for example

yes

**Pull request merged**